document/FileFormat.mdk

```diff
@@ -2385,7 +2390,7 @@ containing a message relevant to this call stack.
 
 A `stack` object shall contain a property named `frames` whose value is an array of one or more `stackFrame` objects (&sect;[#stackFrame]).
 This array shall include every function call in the stack for which the tool has information,
-and the entries that are present shall occur in chronological order with the most recent (innermost) call first and the least recent (outermost) call last.
+and the entries that are present shall occur in nesting order with the most recent (innermost) call first and the least recent (outermost) call last.
```

Comment:
Meh ;-) I really didn't like "chronological" order when I re-read this, and I thought "nesting" was clear, but I'm not adamant about it. I'll change it back.

---
In reply to: [72723785](https://github.com/sarif-standard/sarif-spec/pull/255#discussion_r72723785) [](ancestors = 72723785)